tests/test_template_funcs.py

```diff
@@ -241,6 +242,20 @@ def test_date_from_datetime(self, write_row):
         generate(StringIO(yaml), {}, None)
         assert write_row.mock_calls[0][1][1]["a"] == "2012-01-01"
 
+    def test_now_variable(self, generated_rows):
+        yaml = """
+        - object : A
+          fields:
+            a: ${{now}}
+            b: ${{now}}
+        """
+        generate(StringIO(yaml), {}, None)
+        assert datetime.fromisoformat(generated_rows.table_values("A", 0, "a"))
```

Comment:
This test seems a little imprecise -- it's verifying that `now` generates _a_ date, but it doesn't verify that it sets the correct date. Would it be worth taking the time to mock out datetime.now?

Reply:
@boakley Fixed in 9329a0d8cc83c66eb0a5d97414124e7c33b36384